gammapy/analysis/core.py

```diff
@@ -18,6 +18,7 @@
     FluxPointsEstimator,
     ReflectedRegionsBackgroundMaker,
     SpectrumDatasetMaker,
+    SpectrumDataset
```

Comment:
Run `make polish` before committing (or before merging PRs), to keep formatting clean in gammapy master, and `make polish` available and useful for everyone, to not have to worry about committing unrelated changes.